scylla-artifacts.py

```diff
@@ -263,6 +247,20 @@ def __init__(self, sw_repo=None, mode='ci'):
         self.log = logging.getLogger('avocado.test')
         self.srv_manager = ScyllaServiceManager()
 
+    def try_report_uuid(self):
+        uuid_path = '/var/lib/scylla-housekeeping/housekeeping.uuid'
+        mark_path = '/var/lib/scylla-housekeeping/housekeeping.uuid.marked'
+        cmd = 'curl "https://i6a5h9l1kl.execute-api.us-east-1.amazonaws.com/prod/check_version?uu=%s&mark=scylla"'
+        wait.wait_for(lambda: os.path.exists(uuid_path), timeout=30, step=5,
+                      text='Waiting for housekeeping.uuid generated')
+
+        if os.path.exists(uuid_path) and not os.path.exists(mark_path):
+            with open(uuid_path) as uuid_file:
+                uuid = uuid_file.read().strip()
+            self.log.debug('housekeeping.uuid is %s' % uuid)
```

Comment:
Nitpick, remember you can use commas instead of `%` to pass args to the logging function.